docs/develop/init.md

```diff
@@ -0,0 +1,58 @@
+# Getting Started
+
+You are now ready to learn how to create, customize and iterate on your own
+InvenioRDM instance. The previous section showed you what is the equivalent of
+a deploy compilation when working with a compiled language: it gets you the final
+result, but you wouldn't do that when developing.
+
+When working on your InvenioRDM application, you will want to use the commands
+and the workflow we explain in these pages.
+
+## Initialize your file system
```

Comment:
Ah you noticed my slyness here :hocho: ;) Because, the transition from running locally to running fully in containers hasn't been worked out (see https://github.com/inveniosoftware/invenio-cli/issues/108), I am making the reader create a new project. But I am also positioning this as a way to make sections independent; it was mentioned on the Home page. Finally, repeating commands seen before breeds familiarity with the CLI tool in the developer.